CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ### 2.0.0 (Next)
 
 * [#38](https://github.com/mongoid/mongoid-scroll/pull/38): Allow to reverse the scroll - [@GCorbel](https://github.com/GCorbel).
+* [#42](https://github.com/mongoid/mongoid-scroll/pull/42): Add a cursor to go to the first page - [@GCorbel](https://github.com/GCorbel).
```

Comment:
```suggestion
* [#42](https://github.com/mongoid/mongoid-scroll/pull/42): Add a `first` cursor - [@GCorbel](https://github.com/GCorbel).
```

I think for #38 above we should say "Add a `previous` cursor" too?

Reply:
I changed.